src/cli/miner_cli_info.erl

```diff
@@ -229,13 +231,106 @@ info_region(["info", "region"], [], []) ->
+        undefined ->
+            #{ onboarding_key := MinerOnboardingKey } = miner_keys:keys(),
+            MinerOnboardingKey;
+        ProvidedKey ->
+            ProvidedKey
+    end,
+    PayerOutputOnly = proplists:is_defined(just_payer, Flags),
+    
+    case OnboardingKey of
+        undefined ->
+            error_message("This miner has no onboarding key, no onboarding info available.");
+        OK ->
+            case onboarding_info_for_key(OK) of
+                {ok, OnboardingInfo} ->
+                    clique_status_for_onboarding_info(OnboardingInfo, PayerOutputOnly);
+                notfound ->
+                    error_message("No onboarding info found for this miner's onboarding key.");
+                {error, Code} ->
+                    error_message(io_lib:format("Onboarding server unexcpetedly returned code ~p.", [ Code ]))
+            end
+    end.
+
+%%
+%% Onboarding information is generally divided into two parts: information
+%% specific to this miner ("MinerData") and information about its manufacturer
+%% ("MakerData").
+%%
+-spec onboarding_info_for_key(string()) -> {ok, {map(), map()}} | notfound | {error, non_neg_integer()}.
+onboarding_info_for_key(OnboardingKey) ->
+    Url = ?ONBOARDING_API_URL_BASE ++ "/hotspots/" ++ OnboardingKey, 
+    case get_api_json_as_map(Url) of
+        {ok, OnboardingResult} ->
+            OnboardingData = maps:get(<<"data">>, OnboardingResult),
+            MakerData      = maps:get(<<"maker">>, OnboardingData),
+            IsMinerData = fun (Key, _Value) -> Key =/= <<"maker">> end,
+            MinerData = maps:filter(IsMinerData, OnboardingData),
+            {ok, {MinerData, MakerData}};
+        notfound -> notfound;
+        {error, Code} -> {error, Code}
+    end.
+
+-spec clique_status_for_onboarding_info({map(), map()}, boolean()) -> list().
+clique_status_for_onboarding_info({MinerData, MakerData}, PayerOutputOnly) ->
+    case PayerOutputOnly of
+        true -> 
+            PayerAddress = maps:get(<<"address">>, MakerData),
+            PayerAddressString = binary_to_list(PayerAddress),
+            [ clique_status:text(PayerAddressString) ];
+        false -> full_status_for_onboarding_info(MinerData, MakerData)
+    end.
+
+-spec full_status_for_onboarding_info(map(), map()) -> list().
+full_status_for_onboarding_info(MinerData, MakerData) ->
+    MinerDisplay = maps:to_list(MinerData),
+    MakerDisplay = maps:to_list(MakerData),
+    [
+        clique_status:text("\n********************\nGeneral Manufacturing Info\n********************\n"),
+        clique_status:table(lists:map(fun tuple_to_table_row/1, MinerDisplay)),
+        clique_status:text("\n********************\nManufacturer Info\n********************\n"),
+        clique_status:table(lists:map(fun tuple_to_table_row/1, MakerDisplay))
+    ].
+
 %%
 %% info summary
 %%
 
 info_summary_cmd() ->
     [
-     [["info", "summary"], [], [], fun info_summary/3],
```

Comment:
perhaps this is the GH diff display, but why was this line removed?

Reply:
This line wasn't doing anything. It essentially specifies that you can run `info summary` without any additional arguments. However, the line below it specifies that you can run `info summary` without any additional arguments _or_ with the flag `-e|--error_count`.